test-app/Makefile

```diff
@@ -88,6 +88,10 @@ ifeq ($(TARGET),stm32h7)
   LSCRIPT_TEMPLATE=ARM-stm32h7.ld
 endif
 
+ifeq ($(TARGET),stm32c0)
+  LSCRIPT_TEMPLATE=ARM-stm32c0.ld
+endif
+
```

Comment:
At line 210 below there is a section to force enable `NVM_FLASH_WRITEONCE` for G0. I think this should be done for the C0 also.